frontend/src/utils/Auth.js

```diff
@@ -0,0 +1,45 @@
+export const BASE_URL = 'https://api.yonatanreed.mooo.com';
+// https://api.yonatanreed.mooo.com
+// http://localhost:3000
+
+function processResponse(res) {
+  return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+}
+
+export const authorize = (email, password) => {
+  console.log('auth.js authorize');
+
+  return fetch(`${BASE_URL}/signin`, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ email, password }),
+  }).then(processResponse);
```

Comment:
**Might be better**

You can make a special method for fetching and checking responses **not to duplicate** it in every request:

```js

  function request(url, options) {
    return fetch(url, options).then(processResponse)
  }

```

And now you can safely replace `fetch` with `request` and remove repeated checking. And you don’t need to change anything else.